stko/optimizers/macromodel.py

```diff
@@ -549,9 +549,11 @@ class MacroModelForceField(MacroModel):
 
     Optimisation of `long bonds` only within
     :class:`stk.ConstructedMolecule` is possible with
-    `restricted=True`. This fixes all other bonds. Therefore, if the
-    molecule is not a `ConstructedMolecule`, no positions will be
-    optimized.
+    `restricted=True`.
+    Generally, this means any bonds created during the construction
```

Comment:
This seems incorrect? Should it not be:

"Generally, this means only bonds created during the construction process will be optimized."